src/main/java/com/gooddata/md/visualization/VOPopMeasureDefinition.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2018, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.md.visualization;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+import com.gooddata.executeafm.ObjQualifier;
+
+import static com.gooddata.md.visualization.VOPopMeasureDefinition.NAME;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonRootName(NAME)
+public class VOPopMeasureDefinition extends com.gooddata.executeafm.afm.PopMeasureDefinition {
```

Comment:
Pls make those "VO" classes deprecated, create an issue saying we wanna to unify it and reference that issue in deprecation comment